apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue

```diff
@@ -38,48 +38,60 @@ let search = computed(() => {
 
 const query = computed(() => {
   return `
-  query Networks($filter:NetworksFilter, $orderby:Networksorderby){
-    Networks(filter:$filter  orderby:$orderby) {
-      id
-      name
-      acronym
-      description
-      logo {
+  query Catalogues($filter:CataloguesFilter){
+    Catalogues(filter:$filter) {
+      network {
         id
-        size
-        extension
-        url
+        name
+        acronym
+        description
+        logo {
+          id
+          size
+          extension
+          url
+        }
       }
+      type {name}
     }
-    Networks_agg (filter:$filter){
+    Catalogues_agg (filter:$filter){
       count
     }
   }
   `;
 });
 
-const orderby = { acronym: "ASC" };
-
 const filter = computed(() => buildQueryFilter(filters, search.value));
 
 let graphqlURL = computed(() => `/${route.params.schema}/api/graphql`);
 const { data, pending, error, refresh } = await useFetch(graphqlURL.value, {
-  key: `networks`,
+  key: `catalogues`,
   baseURL: config.public.apiBase,
   method: "POST",
   body: {
     query,
-    variables: { orderby, filter },
+    variables: { filter },
```

Comment:
why is the orderby removed ?

Reply:
a check , this removes the usage , maybe also remove declaration to avoid confusion ( maybe this has been done already)